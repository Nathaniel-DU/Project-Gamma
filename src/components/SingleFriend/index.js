import React from "react";
import DeleteFriend from '../DeleteFriend';

export default ({name, location, userId}) => {
    return (
        <div>
            <b><DeleteFriend userId={userId}/>{name} </b> 
            <br/>
            {location}
            <hr/>
        </div>
    )
}
