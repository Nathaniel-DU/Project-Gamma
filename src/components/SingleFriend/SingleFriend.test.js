import React from 'react';
import { shallow } from 'enzyme';
import SingleFriend from './index';

describe('SingleFriend', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<SingleFriend debug />);
        expect(component).toMatchSnapshot();
    })
})