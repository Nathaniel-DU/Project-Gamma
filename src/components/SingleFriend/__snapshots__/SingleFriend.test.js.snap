// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleFriend should render correctly in "debug" mode 1`] = `
<div>
  <b>
    <DeleteFriend />
     
  </b>
  <br />
  <hr />
</div>
`;
